packages/terra-form-field/src/Field.jsx

```diff
@@ -166,6 +166,18 @@ const Field = (props, { intl }) => {
       ) : null
   );
 
+  const content = React.Children.map(children, (child) => {
```

Comment:
Could we add a jest test around this case?

Reply:
Updated: https://github.com/cerner/terra-core/pull/2398/commits/c92b018fe53f92d1a332a1afdbf29f9c6a621d66